### DISCUSSION TITLE
urlopen error

### DISCUSSION THREAD

**Discussion by qixiaoye** [pyg-team/pytorch_geometric#3546] [Q&A]
I have some trouble when the program which is running in GPU download the dataset from [https://ls11-www.cs.tu-dortmund.de/staff/morris/graphkerneldatasets](url)
![image](https://user-images.githubusercontent.com/37823459/142835519-f1e599af-ab9a-47ca-ac36-7513595e8bc0.png)
here is the error
[
![image](https://user-images.githubusercontent.com/37823459/142835926-5f9d1971-dd16-4c55-b91e-c07ae5968ae3.png)
](url)

thank you for your time

**Comment by rusty1s** [comment]
Which dataset are you trying to download? The `name` passed to `TUDataset` needs to exactly match the ones on the website.
